src/test/java/org/fundacion/jala/converter/core/javacompiler/JavaCompilerTest.java

```diff
@@ -7,7 +7,7 @@
 
 public class JavaCompilerTest {
 
-    @Test(expected = CompilerException.class)
+    @Test(expected = NullPointerException.class)
```

Comment:
Maybe change NullPointerException to CompilerException, this change don't break unit test

Reply:
If I made that change unit test will break, I will change this to CompilerException when we all finish our unit tests